### ISSUE TITLE
Unable to uninstall this SHIT!!

### ISSUE THREAD

**Issue by polnaidenov** [valinet/ExplorerPatcher#1636]
The explorer patcher start crashed. how i can uninstall it completely? 
I send few e-mails to the developer, but no feedback

**Comment by Vipin2003** [comment]
Developer is busy with his stuffs...Do you have access to explorer??

**Comment by Aetopia** [comment]
Just uninstall via Windows Settings or the control panel.

**Comment by uepsie** [comment]
Why would you want to install SHIT at all? :-)

**Comment by pyrates999** [comment]
https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

**Comment by zzj0402** [comment]
> The explorer patcher start crashed. how i can uninstall it completely? I send few e-mails to the developer, but no feedback

You DIDN'T PAY to have this open-source software. Nobody is obligated to work for you for free. If you are so desperate, code it yourself. Watch your language.

**Comment by influential-eliot** [comment]
Hey, @polnaidenov, I recently had to uninstall (_sad times, Win 11 is unusable without it_) here's what I did:

1. Located the downloaded 'install' file (_usually called `ep_setup.exe`_)
2. Opened an Admin PowerShell applet in that location
3. Used this command to copy it: `Copy-Item "ep_setup.exe" -Destination "ep_uninstall.exe"`
4. Ran this command to uninstall: `.\ep_uninstall`

Hope that this helps, also hope that this application gets sorted as it's a necessity for using 11.
